src/components/ThoughtForm.jsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import styles from "./ThoughtForm.module.css";
+
+const ThoughtForm = ({ addThought }) => {
+  const maxLength = 140;
+  const minLength = 5;
+  const [message, setMessage] = useState("");
+  const [error, setError] = useState("");
+
+  const handleTextChange = (event) => {
+    setMessage(event.target.value);
+    if (error) setError("");
+  };
+
+  const getWordCountText = () => {
+    const typedLength = message.length;
+    const remainingLength = maxLength - typedLength;
+
+    let wordCountMessage;
+    if (remainingLength >= 0) {
+      wordCountMessage = `You have typed ${typedLength} letters, you still can type ${remainingLength} letters.`;
```

Comment:
I like this friendly way of helping the user to keep track of the character count, not just 0/140 ☺️